cmd/lncli/commands.go

```diff
@@ -1190,27 +1190,37 @@ mnemonicCheck:
 				len(cipherSeedMnemonic), 24)
 		}
 
-		// Additionally, the user may have a passphrase, that will also
-		// need to be provided so the daemon can properly decipher the
-		// cipher seed.
-		fmt.Printf("Input your cipher seed passphrase (press enter if " +
-			"your seed doesn't have a passphrase): ")
-		passphrase, err := terminal.ReadPassword(int(syscall.Stdin))
-		if err != nil {
-			return err
-		}
+		var passphrase []byte
+		var passphraseConfirmation []byte
```

Comment:
The style in most of the project groups vars together for initialisation:
```
var (
    passphrase             []byte
    passphraseConfirmation []byte
)
```

Reply:
Also not sure if it makes sense to move the vars under the comments? 
```
// we infinite loop passphrase to give user multiple chances to type in
// passphrase correctly
var passphrase []byte
var passphraseConfirmation []byte
for {
```

Or have it's own comments? 
```
// We will store the user input passphrase and passphraseConfirmation
// in the slice of bytes and compare. If bool statement passes then the
// infinite loop will be broken
var (
    passphrase             []byte
    passphraseConfirmation []byte
)
```

Reply:
@ccdle12 i've made the style changes you request, hopefully i got/understood what it was your were looking for. I don't know if you want me to rebase the branch and squash commits to tidy up the git history. Let me know.

Reply:
@cbarraford 

It looks good, in terms of style and formatting I've mainly just followed this and the code styling in master: https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#CodeDocumentation

I'm a new contributor so, it's probably not my call for rebase and squash ;)